class ChargifyException(Exception):
    pass


class ChargifyNotFoundException(ChargifyException):
    pass